integration_tests/pages/listContacts.ts

```diff
@@ -19,6 +19,11 @@ export default class ListContactsPage extends Page {
     return this
   }
 
+  clickContactNamesLink(): ListContactsPage {
```

Comment:
does this just click the first one? I guess we are only showing a single contact?

Reply:
I have added the contactId to `date-qa` and and pass it as paramenter